<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AidatEkleDialog</class>
 <widget class="QDialog" name="AidatEkleDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>267</width>
    <height>112</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Aidat Ekle</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="3">
    <widget class="QDateEdit" name="aidat_tarihi">
     <property name="maximumDateTime">
      <datetime>
       <hour>23</hour>
       <minute>59</minute>
       <second>59</second>
       <year>2050</year>
       <month>12</month>
       <day>31</day>
      </datetime>
     </property>
     <property name="minimumDateTime">
      <datetime>
       <hour>0</hour>
       <minute>0</minute>
       <second>0</second>
       <year>2017</year>
       <month>1</month>
       <day>1</day>
      </datetime>
     </property>
     <property name="date">
      <date>
       <year>2019</year>
       <month>1</month>
       <day>1</day>
      </date>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="1" colspan="2">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Aidat Tarihi</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="4">
    <widget class="QLineEdit" name="input_miktar">
     <property name="placeholderText">
      <string>Miktar</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0" colspan="4">
    <widget class="QLabel" name="aidat_text">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="4" column="0" colspan="4">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="btn_aidat_ekle">
       <property name="text">
        <string>Aidat Ekle</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_mail">
       <property name="text">
        <string>Ekle ve Mail At</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
